endpoints/openrtb2/sample-requests/invalid-whole/interstital-bad-perc.json

```diff
@@ -48,6 +48,7 @@
       }
     }
   },
-  "expectedReturnCode": 400,
-  "expectedErrorMessage": "Invalid request: request.device.ext.prebid.interstitial.minwidthperc must be a number between 0 and 100"
+  "expectedReturnCode": 500,
+  "expectedReturnCodeComment": "This should expect a 400 but at the time of unmarshall it was validating the values, as its being a badinput but prebid throwing it as unmarshall error",
```

Comment:
Can we fix it to return a 400?

Reply:
Yes, Before making this change i like to discuss with you or @bsardo.  In this case validation and unmarshalling are coupled together, not sure why we mixed both of them.